contribs/gmf/src/services/themesservice.js

```diff
@@ -51,6 +56,12 @@ gmf.Themes = function($http, treeUrl, isThemePrivateUrl) {
   this.$http_ = $http;
 
   /**
+   * @type {angular.$q}
+   * @private
+   */
+  this.q_ = $q;
```

Comment:
> Do I understand correctly that we shouldn't call our inner variables with a $ for angular objects

Nope. It says that when we create your own service, we should'nt use `$` like for example `$ngeoBgLayerManager`, our prefix is `ngeo`, `$` is the angular prefix.
But here, the variable is a reference to the angular Service, so for a better understanding, we should keep the `$` prefix on the variable name.
Is it clear ?